evalai/utils/challenges.py

```diff
@@ -224,7 +224,8 @@ def display_challenge_phase_list(challenge_id):
     except requests.exceptions.HTTPError as err:
         if (response.status_code in EVALAI_ERROR_CODES):
             validate_token(response.json())
-            echo(style("Error: {}".format(response.json()["error"]), fg="red", bold=True))
+            echo(style("\nError: {}".format(response.json()["error"]), fg="red", bold=True))
```

Comment:
A better way to get the error message would be `response.json().get("error", "{0} error".format(response.status_code))`.

Reply:
@varunagrawal It is repeated throughout the code, wouldn't it be better to solve it as a different issue?